src/react/ReportControls/ReportControls.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, useEffect, useState } from 'react';
+import SelectReportType from './components/SelectReportType';
+import { Features } from './utilities';
+import SelectIssueType from './components/SelectIssueType';
+import CompareSlider from './components/CompareSlider';
+import Filters from './components/Filters';
+import ViewSettings from './components/ViewSettings';
+import { usePrimaryReportType } from './hooks/usePrimaryReportType';
+
+export interface ReportControlsProps {
+  features: Promise<Features>;
+}
+
+export const ReportControls: FC<ReportControlsProps> = ({ features: featuresProp }) => {
+  const [features, setFeatures] = useState<null | Features>(null);
+  const [primaryReportType] = usePrimaryReportType();
+
+  useEffect(() => {
```

Comment:
If this is coming from observable land there's a way to deal with async observables... I'll have to check. If its not, then we should probably use tanstack here